pkg/reconciler/kubernetes/tektonpipeline/metrics.go

```diff
@@ -33,11 +34,6 @@ var (
 		stats.UnitDimensionless)
 )
 
-const (
-	metricsNew     = "NewInstall"
-	metricsUpgrade = "Upgrade"
-)
```

Comment:
How are we handling count of newInstall and upgrade ?

Reply:
we implement https://github.com/tektoncd/operator/blob/b32c133b7da53f614ddd28c26c8838a59ca1c0c4/pkg/reconciler/kubernetes/tektoninstallerset/client/metrics.go#L21
and the package handles emitting metrics for the component